---
layout: publication
title: "Tumor-specific T cells support chemokine-driven spatial organization of intratumoral immune microaggregates needed for long survival"
key: 2022_immuno_therapy_cancer
date:   2022-03-17 12:00:00
permalink: /publications/2022_immuno_therapy_cancer/
type: article
shortname: 
image: 2022_immuno_therapy_cancer.png
image_large: 2022_immuno_therapy_cancer_teaser.png
image_preview: 2022_immuno_therapy_cancer.png

authors:
- Ziena Abdulrahman
- Saskia J. Santegoets
- Gregor Sturm
- Pornpimol Charoentong
- ijsselsteijn
- somarakis
- hollt
- Francesca Finotello
- Zlatko Trajanoski
- Sylvia L. van Egmond
- Dana A. M. Mustafa
- Marij J. P. Welters
- demiranda
- Sjoerd H. van der Burg

journal: Journal for ImmunoTherapy of Cancer
journal-short: Journal for ImmunoTherapy of Cancer
page_start: e004346
page_end: 
chapter:
volume: 10
issue: 2
year: 2022
editor:
publisher:
school:
award:

doi: 10.1136/jitc-2021-004346
publisher-url:

projects:

external-project:

videos:

pdf: 2022_immuno_therapy_cancer.pdf
supplement: 2022_immuno_therapy_cancer_supplemental.pdf
supplements:
code: 

abstract: "Background The composition of the tumor immune microenvironment (TIME) associated with good prognosis generally also predicts the success of immunotherapy, and both entail the presence of pre-existing tumor-specific T cells. Here, the blueprint of the TIME associated with such an ongoing tumor-specific T-cell response was dissected in a unique prospective oropharyngeal squamous cell carcinoma (OPSCC) cohort, in which tumor-specific tumor-infiltrating T cells were detected (immune responsiveness (IR+)) or not (lack of immune responsiveness (IR-)).Methods A comprehensive multimodal, high-dimensional strategy was applied to dissect the TIME of treatment-naive IR+ and IR- OPSCC tissue, including bulk RNA sequencing (NanoString), imaging mass cytometry (Hyperion) for phenotyping and spatial interaction analyses of immune cells, and combined single-cell gene expression profiling and T-cell receptor (TCR) sequencing (single-cell RNA sequencing (scRNAseq)) to characterize the transcriptional states of clonally expanded tumor-infiltrating T cells.Results IR+ patients had an excellent survival during 10 years follow-up. The tumors of IR+ patients expressed higher levels of genes strongly related to interferon gamma signaling, T-cell activation, TCR signaling, and mononuclear cell differentiation, as well as genes involved in several immune signaling pathways, than IR- patients. The top differently overexpressed genes included CXCL12 and LTB, involved in ectopic lymphoid structure development. Moreover, scRNAseq not only revealed that CD4+ T cells were the main producers of LTB but also identified a subset of clonally expanded CD8+ T cells, dominantly present in IR+ tumors, which secreted the T cell and dendritic cell (DC) attracting chemokine CCL4. Indeed, immune cell infiltration in IR+ tumors is stronger, highly coordinated, and has a distinct spatial phenotypical signature characterized by intratumoral microaggregates of CD8+CD103+ and CD4+ T cells with DCs. In contrast, the IR- TIME comprised spatial interactions between lymphocytes and various immunosuppressive myeloid cell populations. The impact of these chemokines on local immunity and clinical outcome was confirmed in an independent The Cancer Genome Atlas OPSCC cohort.Conclusion The production of lymphoid cell attracting and organizing chemokines by tumor-specific T cells in IR+ tumors constitutes a positive feedback loop to sustain the formation of the DC{\textendash}T-cell microaggregates and identifies patients with excellent survival after standard therapy.Data are available in a public, open access repository. Data are available upon reasonable request. All data relevant to the study are included in the article or uploaded as supplementary information."

---
