---
layout: publication
title: "Comparative transcriptomics reveals human-specific cortical features"
key: 2022_great_apes
date:   2023-10-12 20:00:00
permalink: 
redirect_from: /publications/2022_great_apes/
type: article

shortname: 

image: 2022_great_apes.png
image_large: 2022_great_apes_teaser.png
image_preview: 2022_great_apes.png


authors:
- Nikolas L. Jorstad
- Janet H.T. Song
- David Exposito-Alonso
- Hamsini Suresh
- Nathan Castro-Pacheco
- Fenna M. Krienen
- Anna Marie Yanny
- Jennie Close
- Emily Gelfand
- Kyle J. Travaglini
- basu
- Marc Beaudin
- Darren Bertagnolli
- Megan Crow
- Song-Lin Ding
- eggermont
- Alexandra Glandon
- Jeff Goldy
- kroes
- Brian Long
- Delissa McMillen
- Trangthanh Pham
- Christine Rimorin
- Kimberly Siletti
- Saroja Somasundaram
- Michael Tieu
- Amy Torkelson
- Guoping Feng
- William D Hopkins
- hollt
- C. Dirk Keene
- Sten Linnarsson
- Steven A. McCarroll
- lelieveldt
- Chet C. Sherwood
- Kimberly Smith
- Christopher A. Walsh
- Alexander Dobin
- Jesse Gillis
- Ed S. Lein
- Rebecca D. Hodge
- Trygve E. Bakken

journal: Science
journal-short: Science
page_start: eade9516
page_end: 
chapter:
volume: 382
issue:
year: 2023
preprint-year: 2022
editor:
publisher:
school:
award:

doi: 10.1126/science.ade9516

projects:
- cytosplore
external-project:

pdf: 2023_science.pdf

supplements:
    - name: Collection Intro
      link: 2023_science.pdf
      icon: pdf
    - name: bioRxiv
      abslink: https://doi.org/10.1101/2022.09.19.508480
      icon: fas fa-archive
    - name: Software
      abslink: https://viewer.cytosplore.org
      icon: fas fa-globe


abstract: "<b>INTRODUCTION</b></br> The cerebral cortex is involved in complex cognitive functions such as language. Although the diversity and organization of cortical cell types has been extensively studied in several mammalian species, human cortical specializations that may underlie our distinctive cognitive abilities remain poorly understood.</br>
<b>RATIONALE</b></br> Single-nucleus RNA sequencing (snRNA-seq) offers a relatively unbiased characterization of cellular diversity of brain regions. Comparative transcriptomic analysis enables the identification of molecular and cellular features that are conserved and specialized but is often limited by the number of species analyzed. We applied deep transcriptomic profiling of the cerebral cortex of humans and four nonhuman primate (NHP) species to identify homologous cell types and human specializations.</br>
<b>RESULTS</b></br> We generated snRNA-seq data from humans, chimpanzees, gorillas, rhesus macaques, and marmosets (more than 570,000 nuclei in total) to build a cellular classification of a language-associated region of the cortex, the middle temporal gyrus (MTG), in each species and a consensus primate taxonomy. Cell-type proportions and distributions across cortical layers are highly conserved among great apes, whereas marmosets have higher proportions of L5/6 IT CAR3 and L5 ET excitatory neurons and Chandelier inhibitory neurons. This strongly points to the possibility that other cellular features drive human-specific cortical evolution. Profiling gorillas enabled discrimination of which human and chimpanzee expression differences are specialized in humans. We discovered that chimpanzee neurons have gene expression profiles that are more similar to those of gorilla neurons than to those of human neurons, despite chimpanzees and humans sharing a more-recent common ancestor. By contrast, glial expression changes were consistent with evolutionary distances and were more rapid than neuronal expression changes in all species. Thus, our data support a faster divergence of neuronal, but not glial, expression on the human lineage. For all primate species, many differentially expressed genes (DEGs) were specific to one or a few cell types and were significantly enriched in molecular pathways related to synaptic connectivity and signaling. Hundreds of genes had human-specific differences in transcript isoform usage, and these genes were largely distinct from DEGs. We leveraged published datasets to link human-specific DEGs to regions of the genome with human-accelerated mutations or deletions (HARs and hCONDELs). This led to the surprising discovery that a large fraction of human-specific DEGs (15 to 40%), and particularly those associated with synaptic connections and signaling, were near these genomic regions that are under adaptive selection.</br>
<b>CONCLUSION</b></br> Our study found that MTG cell types are largely conserved across approximately 40 million years of primate evolution, and the composition and spatial positioning of cell types are shared among great apes. In each species, hundreds of genes exhibit cell typeâ€“specific expression changes, particularly in pathways related to neuronal and glial communication. Human-specific DEGs are enriched near likely adaptive genomic changes and are poised to contribute to human-specialized cortical function."

---
